### PR TITLE
add CORE_DCOS_URL

### PR DESCRIPTION
**Pull request by tamarrow**
So we can parameterize `core.dcos_url`.

### CODE REVIEW

File: bin/start_tests.sh [hunk 1]
```diff
@@ -12,8 +12,8 @@ deactivate
 cd cli
 
 # overwrite core.dcos_url in the dcos.toml
-cp tests/data/dcos.toml $DCOS_CONFIG
-sed -i "s/change.dcos.url/$VBOX_IP/g" $DCOS_CONFIG
+cp tests/data/dcos.toml ${DCOS_CONFIG}
+sed -i "s/change.dcos.url/${CORE_DCOS_URL}/g" ${DCOS_CONFIG}
```
Comment (jsancio) [inline bin/start_tests.sh]: Can we change this to `s/http:\/\/change.dcos.url/${CORE_DCOS_URL}/g` or something like that? I would like to be able to keep it as `dcos_url = http://...` in the `dcos.toml`.

Comment (tamarrow) [inline bin/start_tests.sh]: sure

**Comment by tamarrow** [comment]
changed `sed` separator so that don't have to escape either of the urls

**Comment by tamarrow** [comment]
@mgummelt whats the status on getting the ssh tests to optionally run?

**Comment by mgummelt** [comment]
what do you mean "optionally"?  I'm going to make it configurable, but they'll always run

**Comment by mgummelt** [comment]
what problem are you having?

**Comment by tamarrow** [comment]
We're going to have a build that takes in a dcos_url, but we don't want to have aws credentials, so those tests will always fail.

**Comment by mgummelt** [comment]
I don't understand.  What does AWS have to do with our builds?

**Comment by tamarrow** [comment]
for a random `dcos_url` we won't have credentials to ssh to the nodes

**Comment by mgummelt** [comment]
like I said, I am planning on making the credentials configurable.  what do you mean "we don't want to have aws credentials"?

**Comment by tamarrow** [comment]
We don't want teamcity to have multiple aws credentials. for this purpose we just want to pass over the ssh tests. cc @BenWhitehead

**Comment by mgummelt** [comment]
TC doesn't run against AWS

**Comment by tamarrow** [comment]
where is the dcos-image hosted? In any case, most dcos clusters we're testing against run on aws.

**Comment by mgummelt** [comment]
TC runs a local vagrant.

And I don't think that's true, but if you're running against an AWS cluster, you'll be able to configure the tests to use whatever key you want.

**Comment by tamarrow** [comment]
@mgummelt do you know when the nodes test will be configurable? If its not in the current sprint we can close this pr for now.

**Comment by mgummelt** [comment]
not in current sprint

**Pull request closed without merging**